### PR TITLE
feat: Update Review's importer, remove allocation step | NPG-8040

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
# Description

- Removed allocation step from review's importer, it will be moved as a separate tool later
- Setup test in Earthly

### CODE REVIEW

File: utilities/ideascale-importer/Earthfile [hunk 1]
```diff
@@ -73,6 +73,20 @@ docker:
...
+        --pull postgres:14 \
+        --load migrations:latest=(../../containers/event-db-migrations+docker --data=test) \
+        --service migrations \
+        --allow-privileged
+        RUN --secret IDEASCALE_EMAIL \
+            --secret IDEASCALE_PASSWORD \
+            --secret IDEASCALE_API_TOKEN \
+            EVENTDB_URL="postgres://catalyst-event-dev:CHANGE_ME@localhost/CatalystEventDev" \
+            IDEASCALE_API_URL="https://temp-cardano-sandbox.ideascale.com" \
+            poetry run pytest
```
Comment (kukkok3) [inline utilities/ideascale-importer/Earthfile:91]: I think you should run this command with --no-cache as you are testing an external service @jmgilman?

Comment (jmgilman) [reply utilities/ideascale-importer/Earthfile:91]: I haven't been able to verify if this is strictly necessary yet, but it's not a bad idea to add `--no-cache` to the `RUN` command to make sure it runs every time.

Comment (Mr-Leshiy) [reply utilities/ideascale-importer/Earthfile:92]: updated

**Review by kukkok3 — APPROVED** [review]
LGTM

**Pull request merged**